Lib9c/Action/RewardGold.cs

```diff
@@ -38,7 +38,7 @@ public override void LoadPlainValue(IValue plainValue)
 
         public override IWorld Execute(IActionContext context)
         {
-            context.UseGas(1);
```

Comment:
Is `RewardGold` action no longer gassing?

Reply:
Despite of the line, it was treated as a `BlockAction`, so it did not consume gas actually at the libplanet level before. With changes in how actions are processed in libplanet, keeping that line could potentially cause issues, which is why it has been removed to maintain consistency with previous behavior.

Reply:
Thanks for the explanation!